common/plugins/eu.esdihumboldt.hale.common.align/help/assignBound/help.xhtml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Assign Bound</title>
+</head>
+<body>
+<div id ="help">
+	<h4>Example mapping:</h4>
+	<p>
+		The Assign Bound function requires a source schema element as anchor. The function maps a value to a target property only if a source property is present. The Assign Bound function can be useful when you are working with context conditions, because a value will be assigned only for the source values that have been filtered. The Assign function, in contrast, does not have a source data anchor and applies the assigned value to all target instances. <br>
```

Comment:
The function maps a value --> The function assigns a value?

Reply:
OK, will do.